examples/v0/reporting/get_hotel_ads_performance.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example gets Hotel-ads performance statistics for the 50 Hotel ad
+groups with the most impressions over the last 7 days.
+"""
+
+from __future__ import absolute_import
+
+import argparse
+import six
+import sys
+
+import google.ads.google_ads.client
+
+
+_DEFAULT_PAGE_SIZE = 50
+
+
+def main(client, customer_id, page_size):
+    ga_service = client.get_service('GoogleAdsService')
+
+    query = ('SELECT campaign.id, campaign.advertising_channel_type, '
+             'ad_group.id, ad_group.status, metrics.impressions, '
+             'metrics.hotel_average_lead_value_micros, '
+             'segments.hotel_check_in_day_of_week, '
+             'segments.hotel_length_of_stay '
+             'FROM hotel_performance_view '
+             'WHERE segments.date DURING LAST_7_DAYS '
+             'AND campaign.advertising_channel_type = \'HOTEL\' '
+             'AND ad_group.status = \'ENABLED\' '
+             'ORDER BY metrics.impressions DESC '
+             'LIMIT 50')
+
+    response = ga_service.search(customer_id, query, page_size=page_size)
+
+    try:
+        for row in response:
+            campaign = row.campaign
```

Comment:
another style thing from the other Python examples are these one shot throwaway variables. I think we should move away from this and just use the values directly in the print()

Reply:
I'm impartial on this - my main concern is consistency so if we want to change this style let's do it for all the examples separately.

Reply:
Doing separately is fine. I'm not a fan on one shot throwaway variables too, so I agree with David on this.